src/types/view/index.ts

```diff
@@ -116,10 +116,31 @@ export interface ViewWorkflowStepClosedAction extends ViewClosedAction {
   };
 }
 
+export interface ViewStateSelectedOption {
+  text: PlainTextElement;
+  value: string;
+}
+
+export interface ViewStateValue {
+  type: string;
+  value?: string | null;
+  selected_date?: string | null;
+  selected_time?: string | null;
+  selected_conversation?: string | null;
+  selected_channel?: string | null;
+  selected_user?: string | null;
+  selected_option?: ViewStateSelectedOption | null;
+  selected_conversations?: string[];
+  selected_channels?: string[];
+  selected_users?: string[];
+  selected_options?: ViewStateSelectedOption[];
+}
+
 export interface ViewOutput {
   id: string;
   callback_id: string;
   team_id: string;
+  app_installed_team_id?: string;
```

Comment:
While doing the tests for this pull request changes, I found this field is missing.